gazebo_state_plugins/src/GazeboObjectInfo.cpp

```diff
@@ -116,7 +138,20 @@ shape_msgs::SolidPrimitive * GazeboObjectInfo::getSolidPrimitive(physics::Collis
             ROS_ERROR("Dynamic cast failed for box shape");
             return NULL;
         }
-        math::Vector3 bb=box->GetSize();
+
+#if GAZEBO_MAJOR_VERSION >= 8
+        gz_math_Vector3d bb=box->Size();
```

Comment:
Also here it would be nice to minimize code duplication by using a getter for `Size()` / `GetSize()`